test/networkd-test.py

```diff
@@ -423,16 +415,15 @@ def test_search_domains(self):
 
         subprocess.check_call(['systemctl', 'start', 'systemd-networkd'])
 
-        if os.path.islink('/etc/resolv.conf'):
-            for timeout in range(50):
-                with open('/etc/resolv.conf') as f:
-                    contents = f.read()
-                if 'search one\n' in contents:
-                    break
-                time.sleep(0.1)
-            self.assertIn('search one two three four five six\n'
-                          '# Too many search domains configured, remaining ones ignored.\n',
-                          contents)
+        for timeout in range(50):
+            with open(RESOLV_CONF) as f:
+                contents = f.read()
+            if 'search one\n' in contents:
+                break
```

Comment:
why do we need this?
later:
```
self.assertRegex(contents, 'search .*one two three four')
```
I have:
```
# This file is managed by systemd-resolved(8). Do not edit.
#
# This is a dynamic resolv.conf file for connecting local clients directly to
# all known DNS servers.
#
# Third party programs must not access this file directly, but only through the
# symlink at /etc/resolv.conf. To manage resolv.conf(5) in a different way,
# replace this symlink by a static file or a different symlink.
#
# See systemd-resolved.service(8) for details about the supported modes of
# operation for /etc/resolv.conf.

nameserver 10.0.2.3
nameserver 192.168.42.1
search localdomain one two three four five
# Too many search domains configured, remaining ones ignored.
```
so, `if 'search one\n' in contents` returns `False` always